tools/jarcat/tar/tar_test.go

```diff
@@ -0,0 +1,105 @@
+package tar
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"io"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const testDataDir = "tools/jarcat/tar/test_data"
```

Comment:
Yup, it would be an issue because of `filepath.Walk`. 

This path relates to the input test data not the temporary test files; I'd prefer to keep the outputs in the test data dir rather them in /tmp though so plz will clean them up afterwards.
Are you sure about it being in-memory? `ioutil.Temp*` calls `os.TempDir` which returns `/tmp` for me, that's not in-memory on Ubuntu AFAICT?